docs/user/alerting/action-variables.asciidoc

```diff
@@ -125,3 +129,193 @@ triggering data was:
+ 
+[source]
+----
+    {{#FormatDate}} {{{timestamp}}} {{/FormatDate}}
+    {{#FormatDate}} {{{timestamp}}} ; UTC {{/FormatDate}}
+    {{#FormatDate}} {{{timestamp}}} ; UTC; YYYY-MM-DD hh:mma {{/FormatDate}}
+    {{#FormatDate}} {{{timestamp}}} ; ; YYYY-MM-DD hh:mma {{/FormatDate}}
+----
+
+The `<time zone>` parameter must be a valid time zone identifier as listed in https://en.wikipedia.org/wiki/List_of_tz_database_time_zones[TZ database time zone names], such as `"America/New_York"`.
+
+The `<date format>` parameter must be a valid date format string as described in the https://momentjs.com/docs/#/displaying/[Moment `format()` documentation].  For example, the date format  `"YYYY-MM-DD hh:mma"` will render in the following format: `"2023-04-24 11:21pm"`.
+
+The date value itself should usually be referenced with triple braces, as some characters in date strings may contain values which are escaped, which would prevent them from being parsed as dates.
+
+[discrete]
+[[mustache-examples]]
+=== Mustache examples
+
+Below is a Mustache template for an email action sent from an elasticsearch query rule.
+
+[source]
+----
+# {{context.title}}
+
+{{#FormatDate}} {{{date}}} ; America/New_York {{/FormatDate}}
+
+{{context.conditions}}
+
+**documents** _[view in Discover]({{{context.link}}})_
+
+| provider | action | duration |
+| -------- | ------ | -------- |
+{{#context.hits}}{{#_source.event}}| {{provider}} | {{action}} | {{#duration}}{{#EvalMath}} round( {{{duration}}} / 1000 / 1000 / 1000 ) {{/EvalMath}} sec{{/duration}} {{^duration}}-n/a-{{/duration}} |{{/_source.event}}
+{{/context.hits}}
+----
+
+Notes:
+
+- `# {{context.title}}` will render the value of the `context.title` variable as a level 1 heading.
+- `{{#FormatDate}} ... {{/FormatDate}}` will render the value of the `date` variable as a formatted date in the America/New_York time zone.
+- `**documents** _[view in Discover]({{{context.link}}})_` show examples of `**bold**`, `_italic_`, and `[text](url)` links
+- The last section is a table with three columns, with one row per element in the `context.hits` array.  From each of those elements, we access the `provider`, `action`, and `duration` fields of the `_source.event` object.  The `duration` field is rendered as a number of seconds, rounded to the nearest second - it's stored as nanoseconds so needs to be divided by a billion to yield seconds.  The duration field is optional, so we use a `{{#duration}} ... {{/duration}}` section to render the duration if it's present, and `-n/a-` if it's not present.
+
+When rendered into Markdown, and then HTML, and viewed in an email client, it will look like this:
+
+[role="screenshot"]
+image:images/email-mustache-template-rendered.png[Email template rendered in an email client]
+
+The template references the following mustache variables:
```

Comment:
> The template references the following mustache variables:

Is there a reason for restating these? They already appear in the example codeblock, so I'm unsure why they need to be restated. If it's worthwhile to keep them, I'd suggest putting this (and the subsequent data codeblock) at the beginning of this section. For example, along the lines of "If you have an {es} query rule that provides the following Mustache variables...you can create an email action that uses the following Mustache template..."